### DISCUSSION TITLE
Simple question

### DISCUSSION THREAD

**Discussion by ProfAero** [Homebrew/discussions#4601] [Everyday usage]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.25
ORIGIN: https://github.com/Homebrew/brew
HEAD: 2ccdb26c45e0ac8c7d4a8c2cc78cd577c28a2707
Last commit: 17 hours ago
Core tap JSON: 27 Jun 05:10 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 14.0.3 build 1403
Git: 2.39.2 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.88.1 => /usr/bin/curl
macOS: 13.4.1-arm64
CLT: 14.3.1.0.1.1683849156
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

I installed the Homebrew package for freeciv, and it seemed to install fine, but I don't see how to start the executable.  When I installed gnuplot, I just type in gnuplot at the command line, and it fires up just fine.  Not so with freeciv.  Tried freeciv, freeciv-gtk3, not sure how to get it working, or where the files are living.  Any advice appreciated.

**Comment by carlocab** [comment]
Try doing
```
ls -la "$(brew --prefix freeciv)/bin"
```
to see the executables installed by the formula. In this case it looks like you're looking for `freeciv-gtk3.22`. We should probably add a symlink so that can be called as just `freeciv-gtk3`.

**Reply by ProfAero** [reply]
Thanks, that got it.  3.22.  I'll set an alias to make it easier to type.  Much appreciated.
